### DISCUSSION TITLE
开源的nsfw鉴黄服务(仅支持png jpg)

### DISCUSSION THREAD

**Discussion by mingchen3398** [lsky-org/lsky-pro#381] [灌水]
刚才看到 https://github.com/lsky-org/lsky-pro/discussions/360 看来是时候把代码贡献出来了,抛个砖先
基于nodejs
项目地址:https://github.com/mingchen3398/nsfwjs-api-appraisal.git
接口地址;https://nsfw.poshir.top/ (测试用,小服务器承受不了太大压力)
如下设置就行
![image](https://user-images.githubusercontent.com/32698103/162387384-4cca110b-178b-4261-a21a-8715354cb30b.png)

**Comment by mingchen3398** [comment]
现在是 https://nsfw.poshir.top/ 了

**Comment by muziqiu2** [comment]
感谢分享，很好用！省的自己再搭建了

**Comment by NB-XX** [comment]
找了好久NSFWjs的api项目，这个最好用，还有贴心的文档。感谢作者。
